study.md

```diff
@@ -66,7 +66,7 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+FIND BY
```

Comment:
`#all` will return all records of a certain type.  `#where` will return everything if no filters are passed into it.  `#find_by` returns at most one record matching it arguments.